spec/lib/aliquot/payment_spec.rb

```diff
@@ -0,0 +1,160 @@
+require 'aliquot-pay'
+
+require 'json'
+require 'openssl'
+
+describe Aliquot::Payment do
+  let(:token) { AliquotPay.generate_token_ecv1(@payment, key, recipient) }
+  let(:token_string) { JSON.unparse(token) }
```

Comment:
I've never seen `JSON.unparse` used; I thought `token.to_json` was the canonical method?

Reply:
Hmm, I really don't know where I got it from, to be honest.

`.to_json` is better.